### ISSUE TITLE
Import field options/values from integrations (specifically MailChimp Groups)

### ISSUE THREAD

**Issue by Kashkin** [verbb/formie#512]
**What are you trying to do?**
When integrating a form with MailChimp's Interest Categories / Groups, the current implementation relies on knowing the unique hexadecimal id for each item, which isn't available outside of the API.

This makes it practically impossible for non-technical users to use.

**What's your proposed solution?**
This likely adds a great deal of complexity behind the scenes that I'm not considering, but...

Allow users to import options from integrations.

Some considerations:

- This would not be auto-updating. If a user make changes in MailChimp (or any other supported integrations) they would need to return to this screen to click the refresh button and save.
- If values are imported, the value should be shown but disabled. Users can still override the name and default options.
- Groups and Tags could be organised in the select using `<optgroup>`.
- The Groups and Tags field should be disabled / hidden until a user has selected a list.

![image](https://user-images.githubusercontent.com/3699310/124419131-17458200-dd9c-11eb-80c6-b5452dc0379b.png)

**Comment by engram-design** [comment]
Out of interest, the "Interest Categories" mapping value does pull in all available values from Mailchimp. Could you not use those?

![image](https://user-images.githubusercontent.com/1221575/140594081-16db70fe-a1dc-4655-8376-201ae51de094.png)

For example with the above, I have 3 testing categories, which I can pick and assign to, for all submissions on that form. 

I suppose the only issue here is that this can't be dynamic, and is set for every submission on the form. For instance, you might want a checkbox or dropdown field for users to be able to pick which "interest" they have, which as you say is not impossible (by inspecting the mapping screen select for the IDs), but woefully impractical for clients.

We may need to introduce a new type of bulk option for integrations to be able to provide option-based fields (radio/checkbox/dropdown).

**Comment by Kashkin** [comment]
Yeah, it is a dynamic checkbox field that allows a subscriber to set their own preferences.

![image](https://user-images.githubusercontent.com/3699310/140664363-e48a1f13-e6f8-4030-819b-0390723d02a0.png)

Bulk creation would definitely be useful!

**Comment by engram-design** [comment]
Added in [4.0.0-beta.5](https://github.com/verbb/formie/releases/tag/4.0.0-beta.5).
